tests/conftest.py

```diff
@@ -62,7 +62,7 @@ def network():
     return _get_network()
 
 
-@pytest.fixture(scope='session')
+@pytest.yield_fixture(scope='session')
```

Comment:
according to pytest documentation we should not use `yield_fixture` (will be deprecated later). `fixture` also support yield statement (https://docs.pytest.org/en/latest/yieldfixture.html).